sdk-springboot/src/main/java/io/dapr/springboot/util/SpringProcessorHelper.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.springboot.util;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Helper for any/all spring related utility jobs required.
+ */
+public class SpringProcessorHelper {
+
+  /**
+   * Method to provide all possible complete routes list fos this post method present in this controller class,
+   * for mentioned topic.
+   *
+   * @param clazz     Controller class
+   * @param method    Declared method for posting data
+   * @param topicName Associated topic name
+   * @return All possible routes for post mapping for this class and post method
+   */
+  public static List<String> getAllCompleteRoutesForPost(Class clazz, Method method, String topicName) {
```

Comment:
Please, move this as a private method in DaprBeanPostProcessor. The reason is that this design will make this method public and part of our contract - in other words, users can depend on this method. Marking it as private makes it clear that is internal use only. Unfortunately, we don't have an internal visibility, only protected, public, private and default.

Reply:
Agreed that this will expose it without need of exposing. Though, just thinking that at least taking out some methods with default access (package private)/ protected may be required, if refactoring a huge class OR using a required design pattern may force us making some extra methods/interfaces public sometimes. BUT, that's not the case with DaprBeanPostProcessor as of now. So, making the suggested change.